cpp/tests/groupby/shift_tests.cpp

```diff
@@ -359,7 +362,63 @@ TEST_F(groupby_shift_string_test, VeryLargeBackwardOffset)
   size_type offset = -1024;
   auto slr         = cudf::make_string_scalar("42");
 
-  test_groupby_shift_string(key, val, offset, *slr, expected);
+  test_groupby_shift_string_single(key, val, offset, *slr, expected);
+}
+
+template <typename T>
+struct groupby_shift_mixed_test : public BaseFixture {
+};
+
+TYPED_TEST_CASE(groupby_shift_mixed_test, FixedWidthTypes);
```

Comment:
To @karthikeyann 's point, we might want to test decimal explicitly.

Reply:
New tests for `fixed_point` type match added.